plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/model/AccumuloSplit.java

```diff
@@ -59,15 +59,15 @@ public AccumuloSplit(
     }
 
     @JsonProperty("ranges")
-    public List<WrappedRange> getWrappedRanges()
+    public List<SerializedRange> getSerializedRanges()
     {
         return ranges;
     }
 
     @JsonIgnore
     public List<Range> getRanges()
     {
-        return ranges.stream().map(WrappedRange::getRange).collect(Collectors.toList());
+        return ranges.stream().map(SerializedRange::deserialize).collect(Collectors.toList());
```

Comment:
Is that called just once per for split processing?

Reply:
It is called once per split when `AccumuloRecordSet` is created